docs/09-Advanced.md

```diff
@@ -425,6 +445,8 @@ Plugins should derive from Chart.PluginBase and implement the following interfac
 	afterDatasetsUpdate: function(chartInstance) { }
 	afterUpdate: function(chartInstance) { },
 
+	configUpdate: function(chartInstance, newConfig) {},
```

Comment:
A plugin notification by this name is missing in `core.controller.js`, I assume that happens in the (missing) public `updateConfig()` method?